packages/paste-libraries/uid/src/index.tsx

```diff
@@ -13,7 +13,8 @@ const useUID = maybeReactUseId !== undefined ? maybeReactUseId : useUIDLegacy;
 const useUIDSeed =
   maybeReactUseId !== undefined
     ? () => {
-        return (seed: string) => `${maybeReactUseId()}-${seed}`;
+        const id = maybeReactUseId();
+        return (seed: string) => `${id}-${seed}`;
```

Comment:
We noticed the functionality for useUIDSeed was simpler than we initially thought. Rather than generating a new id every time the return function is called, it is only meant to generate one id which is consistent with every call of the return method. Only thing that changes is the suffix that gets passed as an argument.